tests/test_etherscan.py

```diff
@@ -137,6 +141,9 @@ def test_get_address_url(ecosystem, network, expected_prefix, address):
         ("ethereum", NETWORKS["ethereum"][1], "ropsten.etherscan.io"),
         ("fantom", NETWORKS["fantom"][0], "ftmscan.com"),
         ("fantom", NETWORKS["fantom"][1], "testnet.ftmscan.com"),
+        # TODO: Uncomment when Alchemy supports Optimism
```

Comment:
Could maybe switch to Infura instead?

Reply:
we need to add it to the plugin and then we can!